cfme/containers/provider/__init__.py

```diff
@@ -355,6 +361,45 @@ def step(self):
         sel.click(InfoBlock('Overview', 'Topology'))
 
 
+class AdHocMetricsView(BaseLoggedInPage):
+    filter_dropdown = SelectorDropdown('uib-tooltip', 'Filter by')
+    apply_btn = form_buttons.apply_filters
```

Comment:
Any reference to web_ui or pytest_selenium stuff is strictly prohibited when using widgetastic.

Besides, the usage is totally wrong here.